Matplotlib_common/plot_profs.py

```diff
@@ -19,7 +19,8 @@ def plot_profiles(var_list, plot_iter, nplotx, nploty, axarr, xscaledict, yscale
       print file_name
     #  try:
       profiles_file = open(file_name, "r")
-      my_pos = read_my_var(profiles_file, "position")
+      ## my_pos = read_my_var(profiles_file, "position")
+      my_pos = read_my_var(profiles_file, "time")
```

Comment:
When plotting profiles, my_pos is altitude, not time.
If you want to change from position to time in plot_series, please change from position to altitude in plot_profs.
That may require additional changes in matplotlib code.

Reply:
still to be done